tests/src-android/cgeo/geocaching/storage/DataStoreTest.java

```diff
@@ -97,6 +97,66 @@ public static void testStoredLists() {
         }
     }
 
+    public static void testMoveToList() {
+
+        int listId1 = StoredList.STANDARD_LIST_ID;
```

Comment:
Question in general: how do work with code review findings, that might also be applicable to other, unchanged parts of the code? Do you usually fix those, too, or touch other code only if it is related to a bug or feature?